cli/azd/pkg/tools/azcli.go

```diff
@@ -164,6 +170,9 @@ type AzCliResourceOperationProperties struct {
 	TargetResource        AzCliResourceOperationTargetResource `json:"targetResource"`
 	StatusCode            string                               `json:"statusCode"`
 	StatusMessage         AzCliDeploymentStatusMessage         `json:"statusMessage"`
+	// While the deployment is in progress, this timestamp effectively represents "InProgressTimestamp".
```

Comment:
by deployment you mean provision, right?

The comment made me think that there could be two usages for the same field, but this is just the value of time when the resource was read from Azure, right?

Reply:
I'll reword as "operation" since that's the context we're in.

I think this is a design flaw of the ARM API, this field is overloaded with two semantic meanings. I'll update the comment for clarity:
1. Azure updates this timestamp periodically when the operation is in progress, so in effect, this is the in-progress timestamp.
2. When the operation succeeds, the timestamp is frozen and thus would represent the end time of the deployment.